hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/NameNodeLayoutVersion.java

```diff
@@ -89,7 +89,8 @@ public static boolean supports(final LayoutFeature f, final int lv) {
     APPEND_NEW_BLOCK(-62, -61, "Support appending to new block"),
     QUOTA_BY_STORAGE_TYPE(-63, -61, "Support quota for specific storage types"),
     ERASURE_CODING(-64, -61, "Support erasure coding"),
-    EXPANDED_STRING_TABLE(-65, -61, "Support expanded string table in fsimage");
+    EXPANDED_STRING_TABLE(-65, -61, "Support expanded string table in fsimage"),
+    NVDIMM_SUPPORT(-66, -66, "Support NVDIMM storage type");
```

Comment:
Is there any reason to change the `minCompatLV`  to `-66`?

Reply:
As the comment above said: 
     If the feature cannot satisfy compatibility with any prior version, then set its minimum compatible lqyout version to itself to indicate that downgrade is impossible.

Or maybe we missed something?

Reply:
keep it `-61` for` minCompatLV` itself.
More details [here](https://github.com/apache/hadoop/pull/2377#pullrequestreview-518282561)

Reply:
I will change the miniCompatLV to -61, thanks for your clarification.